packages/components/tests/dummy/app/templates/components/accordion.hbs

```diff
@@ -0,0 +1,348 @@
+          <:toggle>Another item two</:toggle>
+          <:content>
+            <Hds::Accordion>
+              <Hds::Accordion::Item>
+                <:toggle>Item one nested</:toggle>
+                <:content>
+                  <Shw::Placeholder @text="generic content" @height="40" @background="#eee" />
+                </:content>
+              </Hds::Accordion::Item>
+              <Hds::Accordion::Item>
+                <:toggle>Item two nested</:toggle>
+                <:content>
+                  <Shw::Placeholder @text="generic content" @height="40" @background="#eee" />
+                </:content>
+              </Hds::Accordion::Item>
+            </Hds::Accordion>
+          </:content>
+        </A.Item>
+      </Hds::Accordion>
+    </SF.Item>
+
+    <SF.Item @label="Nested Accordion with isInteractive=false nested in isInteractive=true">
+      <Hds::Accordion as |A|>
+        <A.Item @isOpen={{true}}>
+          <:toggle>Item with isInteractive=true (the default)</:toggle>
+          <:content>
+            <Hds::Accordion>
+              <Hds::Accordion::Item @isInteractive={{false}}>
+                <:toggle>Nested item with isInteractive=false</:toggle>
+                <:content>
+                  <Shw::Placeholder @text="generic content" @height="40" @background="#eee" />
+                </:content>
+              </Hds::Accordion::Item>
+            </Hds::Accordion>
+          </:content>
+        </A.Item>
+      </Hds::Accordion>
+    </SF.Item>
+
+    <SF.Item @label="Nested Accordion with isInteractive=true nested in isInteractive=false">
+      <Hds::Accordion as |A|>
+        <A.Item @isOpen={{true}} @isInteractive={{false}}>
+          <:toggle>Item with isInteractive=false</:toggle>
+          <:content>
+            <Hds::Accordion>
+              <Hds::Accordion::Item>
+                <:toggle>Nested item with isInteractive=true (the default)</:toggle>
+                <:content>
+                  <Shw::Placeholder @text="generic content" @height="40" @background="#eee" />
+                </:content>
+              </Hds::Accordion::Item>
+            </Hds::Accordion>
+          </:content>
+        </A.Item>
+      </Hds::Accordion>
+    </SF.Item>
+
+    <SF.Item @label="With rich content in toggle (HTML header tag)">
+      <Hds::Accordion as |A|>
+        <A.Item>
+          <:toggle>
+            <h4 class="hds-typography-display-300">Header tag example</h4>
+          </:toggle>
+          <:content>
+            <Shw::Placeholder @text="generic content" @height="40" @background="#eee" />
+          </:content>
+        </A.Item>
+      </Hds::Accordion>
+    </SF.Item>
+
+    <SF.Item @label="With interactive content in toggle and in toggled content">
+      <Hds::Accordion as |A|>
+        <A.Item @isInteractive={{false}}>
+          <:toggle>
+            Related text
+            <a href="https://www.hashicorp.com/">A link</a>
+            <Hds::Button @text="HDS Button" @size="small" />
```

Comment:
suggestion:
maybe this could be made more similar to something like this?
<img width="834" alt="screenshot_2819" src="https://github.com/hashicorp/design-system/assets/686239/4cff62e7-f510-45d4-b684-9f8d626985c5">

reference: 
<img width="526" alt="screenshot_2818" src="https://github.com/hashicorp/design-system/assets/686239/c5bcd6b4-3dbc-492c-94b5-556d258f9b96">

Reply:
2 cents tip: trying to have examples that are more similar to production at times helps to uncover limitations in the component that are not obvious at first sight, for example the width not 100% (see comment above)

Reply:
I updated the example to improve it. 

(I wasn't sure of a good example when I added it but had intended to improve it before releasing. Also part of why I just added that overly simple example to start though was that it bothers me to write one-off custom layout styles.)